### PR TITLE
vote: bufferflies as PD committer

### PR DESCRIPTION
**Pull request by nolouch**

### CODE REVIEW

**Review by disksing — APPROVED** [review]

File: votes/0137-bufferflies-as-pd-committer.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# A Vote for bufferflies as PD Committer
+
+## Proposal
+
+[@bufferflies](https://github.com/bufferflies) has been working on `tikv/pd` for more than four months. He has continuously contributed to scheduler、recover、statistics for PD. His pr are listed [here]( https://github.com/tikv/pd/pulls?q=+is%3Apr+author%3Abufferflies+).
```
Comment (tisonkun) [inline votes/0137-bufferflies-as-pd-committer.md:5]:
```suggestion
[@bufferflies](https://github.com/bufferflies) has been working on `tikv/pd` for more than four months. He has continuously contributed to scheduler, recover, statistics for PD. His pull requests are listed [here]( https://github.com/tikv/pd/pulls?q=+is%3Apr+author%3Abufferflies+).
```

@nolouch I'd suggest use English punctuation, and if possible, describe more on why we nominate @bufferflies as committer to build an image to the community.

**Review by tisonkun — COMMENTED** [review]
Leave one comment inline, and it is a reminder that promoting a new committer requests at least 6 days to elapse, i.e., until July 26th.

File: votes/0137-bufferflies-as-pd-committer.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# A Vote for bufferflies as PD Committer
+
+## Proposal
+
+[@bufferflies](https://github.com/bufferflies) has been working on `tikv/pd` for more than four months. He has continuously contributed to the scheduler, recovery, and statistics for PD, and His main contribution is to simulate scheduling behaviors with [MathLab](https://github.com/bufferflies/tidb-simulink), which is a challenging task, which establish control modeling for scheduling and can help to find optimal ideas.
```
Comment (tisonkun) [inline votes/0137-bufferflies-as-pd-committer.md:5]:
```suggestion
[@bufferflies](https://github.com/bufferflies) has been working on `tikv/pd` for more than four months. He has continuously contributed to the scheduler, recovery, and statistics for PD.

His main contribution is implementing simulate scheduling behaviors with [MathLab](https://github.com/bufferflies/tidb-simulink), which establishes control modeling for scheduling and can help to find optimization ideas.
```

**Review by JmPotato — APPROVED** [review]

**Review by lhy1024 — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]
+1 (non-binding)

**Review by rleungx — APPROVED** [review]
+1

**Comment by tisonkun** [comment]
@nolouch @rleungx 6 days elapsed and we have already 2 binding votes. It is good to merge.

**Comment by nolouch** [comment]
Let's merge it!

**Pull request merged**